example/ios/Podfile.lock

```diff
@@ -0,0 +1,51 @@
+PODS:
+  - GoogleMaps (2.0.1):
+    - GoogleMaps/Maps (= 2.0.1)
+  - GoogleMaps/Base (2.0.1)
+  - GoogleMaps/Maps (2.0.1):
+    - GoogleMaps/Base (= 2.0.1)
+  - React/Core (0.32.1):
```

Comment:
Elsewhere in the codebase expects 0.32.0 to be installed.  Can you make these 0.32.0?